Bus Scheduler/src/components/tripSearch/store/types.ts

```diff
@@ -0,0 +1,23 @@
+
+export type TripListType =
+  {
+    data: {
+      resultCount: number;
+      trips: { busId: string; id: number; busType: string; totalSeats: number; seatType: string }[];
+      message?: string;
+      success?: boolean;
+    }
+  };
+
+export type TripListContextType = {
+  data: TripListType;
+  actions: TripSearchActionsType
+};
+
+export type TripSearchActionsType = {
+  listTrips: (payload: TripListType) => void
+}
+
+export enum ActionKind {
```

Comment:
can we create a new file enum and move it there

Reply:
Done